client/my-sites/plans/jetpack-plans/product-grid/utils.ts

```diff
@@ -64,6 +65,10 @@ export const getPlansToDisplay = ( {
 	return plansToDisplay;
 };
 
+const removeAddons = (
+	product: SelectorProduct | null // eslint-disable-next-line @typescript-eslint/no-explicit-any
+) => ! JETPACK_BACKUP_ADDON_PRODUCTS.includes( product?.productSlug as any );
```

Comment:
Possibly there's even a better way of doing this, but here's my suggestion:

```suggestion
const removeAddons = (
	product: SelectorProduct | null // eslint-disable-next-line @typescript-eslint/no-explicit-any
) => product: SelectorProduct | null ) =>
	product &&
	! ( JETPACK_BACKUP_ADDON_PRODUCTS as ReadonlyArray< string > ).includes( product?.productSlug );
```

This way we don't need to type it as `any`.

Reply:
That seems like the right way to do it. Thanks for the suggestion